umiCollapse.wdl

```diff
@@ -21,6 +22,8 @@ workflow umiCollapse {
         String pattern1 
         String pattern2
         String reference
+        Boolean doBamQC = false
+        Boolean provisionBam = true
     }
 
     Map[String, GenomeResources] resources = {
```

Comment:
note that there is a  comma after mm10 block

Reply:
removed